commands/proofs/tx.go

```diff
@@ -22,6 +22,10 @@ data to other peers as needed.
 }
 
 func doTxQuery(cmd *cobra.Command, args []string) error {
+	if err := commands.RequireInit(cmd); err != nil {
```

Comment:
That was my other thought. I'd rather that `RequireInit(runQuery)` piece than copying this everywhere :)